cosmic-panel-config/Cargo.toml

```diff
@@ -13,7 +13,7 @@ ron = "0.8.0"
 serde = { version = "1.0.152", features = ["derive"] }
 tracing = "0.1.37"
 wayland-protocols-wlr = { version = "0.1.0", features = ["server"], optional = true}
-cosmic-config = { git = "https://github.com/pop-os/libcosmic", rev = "f06a81c" }
+cosmic-config = { git = "https://github.com/pop-os/libcosmic", rev = "31f7e97" }
```

Comment:
The revision should only be defined for the binary project, since cosmic-settings will also be importing libcosmic from a different commit